x/interchainstaking/keeper/callbacks_test.go

```diff
@@ -1606,6 +1606,34 @@ func (suite *KeeperTestSuite) TestDelegationAccountBalanceCallback() {
 		suite.NoError(err)
 	})
 }
+func (suite *KeeperTestSuite) TestPerfBalanceCallback() {
+	suite.Run("perf balance", func() {
+		suite.SetupTest()
+		suite.setupTestZones()
+
+		quicksilver := suite.GetQuicksilverApp(suite.chainA)
+		quicksilver.InterchainstakingKeeper.CallbackHandler().RegisterCallbacks()
+		ctx := suite.chainA.GetContext()
+
+		zone, _ := quicksilver.InterchainstakingKeeper.GetZone(ctx, suite.chainB.ChainID)
+		zone.DepositAddress.IncrementBalanceWaitgroup()
+		zone.WithdrawalAddress.IncrementBalanceWaitgroup()
+		quicksilver.InterchainstakingKeeper.SetZone(ctx, &zone)
+
+		response := sdk.NewCoin("qck", sdk.NewInt(10))
+		respbz, err := quicksilver.AppCodec().Marshal(&response)
+		suite.NoError(err)
+
+		for _, addr := range []string{zone.DepositAddress.Address, zone.WithdrawalAddress.Address} {
```

Comment:
We should probably be passing in the performance address here. Interesting that it works regardless of what address you return a query for; but we probably should check in this fn that we are using the correct address, else the following function could still fail.